packages/core/src/utils/cssVars/cssVars.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Generates a plain object of CSS variable key/value pairs based on the specified `vars`.
+ * @note Filters out falsy values and transforms names to kebab-case.
+ * @example
```

Comment:
Minor style changes:

```suggestion
 * Generates a plain object of CSS variable key/value pairs based on the
 * specified `vars`.
 *
 * @note Filters out falsy values and transforms names to kebab-case.
 *
 * @example
```

Reply:
Changed, thanks!